src/components/Navigation.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './Navigation.module.css';
+
+const Navbar = [
+  { to: '/home', text: 'Home  |' },
+  { to: '/calculator', text: 'Calculator  |' },
+  { to: '/hittheapi', text: 'Quote  |' },
+];
+
+const Nav = () => (
+  <nav className={styles.navigation}>
+    <h1>Math Magicians</h1>
+    <ul className={styles.item}>
+      {Navbar.map((link) => (
+        <li key={link.to}>
+          <NavLink to={link.to} className={styles.links}>
+            {link.text}
+          </NavLink>
+        </li>
+      ))}
+    </ul>
+  </nav>
```

Comment:
- [ ] Please make sure to use React basic styling instead of CSS modules as requested in the project requirements.

![image](https://github.com/mailsg/Math-Magicians/assets/83803414/699ddc4b-412a-4b4e-b4b0-7b1cad6b1ea2)

Reply:
Used React basic styling by importing CSS.